src/Components/Notifications/Notification.tsx

```diff
@@ -38,3 +56,60 @@ export const NotificationFragmentContainer = createFragmentContainer(
     `,
   }
 )
+
+const QUERY = graphql`
+  query NotificationQuery($id: String!) {
+    me {
+      ...Notification_me @arguments(notificationId: $id)
+    }
+  }
+`
+
+export const NotificationQueryRenderer: React.FC = () => {
+  const { state } = useNotificationsContext()
+
+  if (!state.currentNotificationId) {
+    return null
+  }
+
+  return (
+    <SystemQueryRenderer<NotificationQuery>
```

Comment:
**question (non-blocking):** General question re: preferred practice in Force these days — `useLazyLoadQuery` vs `useClientQuery` vs `SystemQueryRenderer`. Is it just up to the developer, or are there reasons to prefer the QR here?

Reply:
I changed the code to use `useLazyLoadQuery` instead of `QueryRenderer` 👍 
I'm not sure about `useClientQuery` and when to use it over `useLazyLoadQuery`. 🤔